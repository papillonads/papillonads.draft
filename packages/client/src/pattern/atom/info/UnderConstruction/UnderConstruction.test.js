import underConstructionDefault from './UnderConstruction.render'

describe('<UnderConstruction />', () => {
  describe('Snaphot', () => {
    test('must match underConstructionDefault', () => {
      expect(global.renderToJSON(underConstructionDefault)).toMatchSnapshot()
    })
  })
})
