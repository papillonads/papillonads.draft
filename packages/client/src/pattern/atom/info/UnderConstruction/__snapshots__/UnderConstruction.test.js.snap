// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UnderConstruction /> Snaphot must match underConstructionDefault 1`] = `null`;
